tests/dragonfly/utility.py

```diff
@@ -15,68 +20,413 @@ def grouper(n, iterable):
         yield chunk
 
 
-BATCH_SIZE = 100
+def eprint(*args, **kwargs):
+    """Print to stderr"""
+    print(*args, file=sys.stderr, **kwargs)
 
 
-def gen_test_data(n, start=0, seed=None):
-    for i in range(start, n):
-        yield "k-"+str(i), "v-"+str(i) + ("-"+str(seed) if seed else "")
+async def wait_available_async(client: aioredis.Redis):
```

Comment:
at what cases this is useful? I mean from what I know await will block the current task until the awaited function will return. So when the iteration will take place?

Reply:
This is for waiting until an instance becomes available for queries (i.e. exits the LOADING state). Its indeed supposed to block all this time, because the test has nothing else to to except wait for the instance to be available for comparing data